server/src/mikro-orm.config.ts

```diff
@@ -3,13 +3,14 @@ import { MariaDbDriver } from "@mikro-orm/mariadb";
 import path from "path";
 import { Save } from "./models/save.model";
 import { User } from "./models/user.model";
+import { ENV } from "./enums/Env";
 
 // The configuration for the ORM - Any Entities added need to be put in here, other than that probably doesn't need to be touched
 // tslint:disable-next-line: no-object-literal-type-assertion
 export default {
   type: "mariadb",
   allowGlobalContext: false,
-  debug: process.env.ENV !== "production" ? true : false,
+  debug: process.env.ENV !== ENV.PROD ? true : false,
```

Comment:
We dont need the ternary here process.env.ENV !== ENV.PROD evaluates to true or false

Reply:
Fixed in [latest commit](https://github.com/bym-refitted/backyard-monsters-refitted/commit/4d306e150f1f7dfb4c92ef94d6f2c0084345cb08)